src/TwoDays.tsx

```diff
@@ -290,13 +304,17 @@ function MessagePoster() {
           return;
         }
 
-        const res = setDoc(
+        if (isNickAction) {
+          const newName = `${messageValue}`.replace("/nick ", "");
```

Comment:
It would be good to `trim()` after `replace` to remove extra spaces around the name.

(Could also add a check to ignore empty names, but maybe it's interesting to allow empty names?)